cfme/common/physical_server_views.py

```diff
@@ -0,0 +1,188 @@
+    """Represents Details page."""
+    properties = SummaryTable(title="Properties")
+    networks = SummaryTable(title="Networks")
+    relationships = SummaryTable(title="Relationships")
+    power_management = SummaryTable(title="Power Management")
+    assets = SummaryTable(title="Assets")
+    firmware = SummaryTable(title="Firmware")
+    network_devices = SummaryTable(title="Network Devices")
+    smart = SummaryTable(title="Smart Management")
+
+
+class PhysicalServerDetailsView(ComputePhysicalInfrastructureServersView):
+    """Main PhysicalServer details page."""
+    breadcrumb = BreadCrumb(locator='.//ol[@class="breadcrumb"]')
+    toolbar = View.nested(PhysicalServerDetailsToolbar)
+    entities = View.nested(PhysicalServerDetailsEntities)
+
+    @property
+    def is_displayed(self):
+        title = "{name} (Summary)".format(name=self.context["object"].name)
+        return (self.in_compute_physical_infrastructure_servers and
+                self.breadcrumb.active_location == title)
+
+
+class PhysicalServerTimelinesView(TimelinesView, ComputePhysicalInfrastructureServersView):
+    """Represents a PhysicalServer Timelines page."""
+
+    @property
+    def is_displayed(self):
+        return (self.in_compute_physical_infrastructure_servers and
+                super(TimelinesView, self).is_displayed)
+
+
+class PhysicalServerManagePoliciesView(BaseLoggedInPage):
+    """PhysicalServer's Manage Policies view."""
+    policies = BootstrapTreeview("protectbox")
+    entities = View.nested(BaseNonInteractiveEntitiesView)
+    save_button = Button("Save")
+    reset_button = Button("Reset")
+    cancel_button = Button("Cancel")
+
+    @property
+    def is_displayed(self):
+        return False
+
+
+class PhysicalServersToolbar(View):
+    """Represents hosts toolbar and its controls."""
+    configuration = Dropdown(text="Configuration")
+    policy = Dropdown(text="Policy")
+    lifecycle = Dropdown(text="Lifecycle")
+    monitoring = Dropdown(text="Monitoring")
+    power = Dropdown(text="Power")
+    identify = Dropdown(text="Identify")
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class PhysicalServerSideBar(View):
+    """Represents left side bar. It usually contains navigation, filters, etc."""
+    pass
+
+
+class PhysicalServerEntitiesView(BaseEntitiesView):
+    """Represents the view with different items like hosts."""
+    @property
+    def entity_class(self):
+        return PhysicalServerEntity().pick(self.browser.product_version)
+
+
+class PhysicalServersView(ComputePhysicalInfrastructureServersView):
+    toolbar = View.nested(PhysicalServersToolbar)
+    sidebar = View.nested(PhysicalServerSideBar)
+    paginator = PaginationPane()
+    including_entities = View.include(PhysicalServerEntitiesView, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        return (self.in_compute_physical_infrastructure_servers and
+                self.title.text in "Physical Servers")
```

Comment:
**Required**

This title check should be an `==` comparison, since filtered views also include the 'Physical Servers' string in their title.
Adding the `Filters` accordion widget to the sidebar view (previous comment) would allow for an additional check that the `ALL (Default)` option is selected and the view is unfiltered.